CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ N/A
 
 ## Changes since v5.0.0
 
+- [#462](https://github.com/oauth2-proxy/oauth2-proxy/pull/462) Allow HTML in banner message (@eritikass).
```

Comment:
This will need moving now to line 22 in the `changes since v5.1.0` section, then I'm happy to merge